.gitignore

```diff
@@ -389,6 +389,9 @@ tests/integration/testdata/buildcmd/Dotnetcore2.0/bin
 tests/integration/testdata/buildcmd/Dotnetcore2.0/obj
 tests/integration/testdata/buildcmd/Dotnetcore2.1/bin
 tests/integration/testdata/buildcmd/Dotnetcore2.1/obj
+tests/integration/testdata/buildcmd/Dotnetcore3.1/bin
+tests/integration/testdata/buildcmd/Dotnetcore3.1/obj
+tests/integration/testdata/invoke/credential_tests/inprocess/dotnet/STS/obj
```

Comment:
does .dotnet 2.1 or 2.0 also have this folder?

Reply:
There is a folder for .NET 2.1 but none for .NET 2.0
I guess it was prior .NET 2.1